### PR TITLE
Add a `short_url` field to beacon responses

### LINKED ISSUE #10: Return the full shortened URL, including host when returning beacon info
At the moment we only return an 'id' and it's expected that the client knows in advance the domain name of the shortener. 

We should return a new field: `short_url` that contains the full URL.

### PR DESCRIPTION
**Pull request by samgiles**
@wilsonpage 

r?

Closes #10

### CODE REVIEW

File: content-service/README.md [hunk 1]
```diff
@@ -85,6 +84,22 @@ TODO: Pagination
 
 Get information about a beacon.
 
+```JS
+{
+  id: <string>,
+  short_url: <string>,
+  channel: <string>,
+  url: <string>,
+  call_to_action: <object>,
```
Comment (wilsonpage) [inline content-service/README.md:93]: Do we just have one now?

File: content-service/scripts/waitforpg.js [hunk 2]
```diff
@@ -2,4 +2,4 @@ const { waitForPg } = require('areyoubeingserved');
 const environment = process.env.ENV || 'dev';
 const dbConfig = require('../database.json')[environment.toLowerCase()];
 
-module.exports = function() { return waitForPg(10, dbConfig)(); };
+module.exports = function() { return waitForPg(30, dbConfig)(); };
```
Comment (wilsonpage) [inline content-service/scripts/waitforpg.js:5]: What is `pg`?

File: content-service/server/database/beacons.js [hunk 3]
```diff
@@ -27,6 +29,41 @@ module.exports = function(knex) {
       });
   }
 
+  function mapDatabaseResponseToApiResponse(beacon) {
+    const location = JSON.parse(beacon.location);
+    const short = shortId.numToShortId(beacon.id);
+    return {
+      id: short,
+      short_url: resolveUrl(SHORT_URL, short),
```
Comment (wilsonpage) [inline content-service/server/database/beacons.js:37]: Bloody love `url.resolve()`, so handy!

File: content-service/server/database/beacons.js [hunk 4]
```diff
@@ -27,6 +29,41 @@ module.exports = function(knex) {
...
+      extra_metadata: JSON.parse(beacon.extra_metadata),
+      location:  {
+        latitude: location.coordinates[1],
+        longitude: location.coordinates[0],
+      },
+      is_virtual: beacon.is_virtual,
+    };
+  }
+
+
```
Comment (wilsonpage) [inline content-service/server/database/beacons.js]: NIT remove line.

File: content-service/server/database/beacons.js [hunk 5]
```diff
@@ -27,6 +29,41 @@ module.exports = function(knex) {
...
+  function batchGetBeaconInfo(channelName) {
```
Comment (wilsonpage) [inline content-service/server/database/beacons.js]: Any reason for this naming? `getAllBeaconsForChannel` seemed to make more sense to me.

File: content-service/config.json [hunk 6]
```diff
@@ -2,5 +2,6 @@
   "api_key": "test",
   "acao_list": [
     "http://localhost:3001"
-  ]
+  ],
+  "short_url": "http://localhost:3002"
```
Comment (wilsonpage) [inline content-service/config.json:6]: How does this get overridden to `https://pm0.io` in production?

File: content-service/README.md (hunk 1 above)
Comment (samgiles) [reply content-service/README.md:93]: No, it's a Map.  But I wanted to start changing from plurals to singulars.  It's a better convention and is generally how database table naming conventions go.  Currently the APIs are 'call_to_action' I think, while the database has a 'calls_to_action' field.  The database schema needs.. updating, it's a little off what the API became.

File: content-service/scripts/waitforpg.js (hunk 2 above)
Comment (samgiles) [reply content-service/scripts/waitforpg.js:5]: Postgres.

File: content-service/config.json (hunk 6 above)
Comment (samgiles) [reply content-service/config.json:6]: The same way `api_key` is overridden at the moment: there is a file on production that isn't checked in with the config altered.

File: content-service/server/database/beacons.js (hunk 5 above)
Comment (samgiles) [reply content-service/server/database/beacons.js]: No particular reason, I'll change it back, these database files are getting unruly anyhow and need a re-org.  Most of the logic happens in these files.

**Comment by wilsonpage** [comment]
ship.ly!

File: content-service/server/database/beacons.js (hunk 5 above)
Comment (samgiles) [reply content-service/server/database/beacons.js]: Turns out this function was unused in this file anyway.

**Pull request merged**